modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributor.java

```diff
@@ -254,7 +254,9 @@ protected String getItemSelectorURL(
 		Group group = _groupLocalService.fetchGroup(groupId);
 
 		if (group == null) {
-			return StringPool.BLANK;
+			ThemeDisplay themeDisplay = getThemeDisplay(httpServletRequest);
+
+			group = themeDisplay.getScopeGroup();
```

Comment:
Good job, @carolmariaabb! I was imaging that a null Group could bring issues to us. 🙂